packages/shared-ui/src/flowHandler/flowHandlerState.ts

```diff
@@ -102,4 +115,20 @@ export class FlowHandlerState {
 
     return userState;
   }
+
+  #checkPasskeyAppendIntervalPassed(intervalParam: string) {
+    const interval = passkeyAppendIntervalMap[intervalParam] ?? 0;
```

Comment:
I think we should translate intervalParam into number using "passkeyAppendIntervalMap" even earlier because that's the job of FlowHandlerConfig right? (it's description says "The FlowHandlerConfig class is responsible for managing the configuration of an authentication flow. It holds the processed version of the information from the project config, and other inputs of the FlowHandler.")
=> IMO we should move this translation to FlowHandlerConfig and it should directly expose a number instead of a string in the passkeyAppendInterval getter.